src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  position: relative;
+  text-decoration: none;
+  color: black;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector). Note that the `box-sizing` is only property allowed to set directly in `*` selector